section2/reflection.md

```diff
@@ -1,29 +1,98 @@
-## Section 2 Reflection
+## Section 2
```

Comment:
Ah! @priyapower I did not save files for those, because I just kept handwritten notes (like, kind of a lot of them...). Happy to add something in here, if needed.

Reply:
Nope, thats perfect :)